sal_interfaces/ATPneumatics/ATPneumatics_Commands.xml

```diff
@@ -44,7 +44,7 @@
         <Subsystem>ATPneumatics</Subsystem>
         <Version>3.7.1</Version>
         <Author/>
-        <Description>Set the desired air pressure for the M2 support system.</Description>
+        <Description>Set the desired air pressure for the M2 support system. Hardware not functional</Description>
```

Comment:
I think the fact that the "Hardware is not functional" should not be added on the xml. Otherwise, we will have to update the xml once the hardware becomes function.

Reply:
Based on discussion with Tiago, the description has been updated to include a reference to a ticket where this functionality will be renamed (since the M2 will never be connected).